utils/ort/src/main/kotlin/storage/S3FileStorage.kt

```diff
@@ -28,6 +28,7 @@ import java.net.URI
 import org.apache.commons.compress.compressors.xz.XZCompressorInputStream
 import org.apache.commons.compress.compressors.xz.XZCompressorOutputStream
 import org.apache.logging.log4j.kotlin.logger
+import org.ossreviewtoolkit.utils.common.collectMessages
```

Comment:
This import also needs an empty line before. I was a bit inaccurate referring to "top-level domains" as both come from "org", but we actually look one level deeper for grouping. And as "org.apache" is different from "org.ossreviewtoolkit", these groups need to be separated by empty lines.

Reply:
Ah, so you did that change, but in the last commit instead of here.

Reply:
Still, the empty line addition needs to be moved to this commit.